src/main/java/net/greatstart/model/User.java

```diff
@@ -1,15 +1,39 @@
 package net.greatstart.model;
 
+import org.hibernate.annotations.LazyCollection;
+import org.hibernate.annotations.LazyCollectionOption;
+import org.springframework.stereotype.Component;
+
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.persistence.Embedded;
+import javax.persistence.Enumerated;
+import javax.persistence.Column;
+import javax.persistence.OneToMany;
+import javax.persistence.EnumType;
 import java.util.List;
 
-public class User {
-    private int id;
+@Component
+@Entity
+@Table(name = "users")
+public class User extends AbstractModel {
     private String name;
     private String password;
     private String email;

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "email" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)